buildenv/jenkins/variables/defaults.yml

```diff
@@ -81,6 +81,10 @@ linux_ppc-64_cmprssptrs_le:
       10: 'hw.arch.ppc64le && sw.os.ubuntu'
       11: 'hw.arch.ppc64le && sw.os.ubuntu'
       next: 'hw.arch.ppc64le && sw.os.ubuntu'
+  build_env:
+    vars:
+      11: 'CC=/usr/bin/gcc-7 CXX=/usr/bin/g++-7'
```

Comment:
`/usr/bin` should be in `PATH`; configure prefers that relative paths are used, as in:
```
CC=gcc-7 CXX=g++-7
```